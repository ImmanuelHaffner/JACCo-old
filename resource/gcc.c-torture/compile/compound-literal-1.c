


struct A { int i; };

void foo()
{
    ((struct A) { 0 }).i += 1;
}
