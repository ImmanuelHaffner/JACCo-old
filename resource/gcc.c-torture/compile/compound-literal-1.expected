0
compound-literal-1.c:4:1 keyword struct
compound-literal-1.c:4:8 identifier A
compound-literal-1.c:4:10 punctuator {
compound-literal-1.c:4:12 keyword int
compound-literal-1.c:4:16 identifier i
compound-literal-1.c:4:17 punctuator ;
compound-literal-1.c:4:19 punctuator }
compound-literal-1.c:4:20 punctuator ;
compound-literal-1.c:6:1 keyword void
compound-literal-1.c:6:6 identifier foo
compound-literal-1.c:6:9 punctuator (
compound-literal-1.c:6:10 punctuator )
compound-literal-1.c:7:1 punctuator {
compound-literal-1.c:8:5 punctuator (
compound-literal-1.c:8:6 punctuator (
compound-literal-1.c:8:7 keyword struct
compound-literal-1.c:8:14 identifier A
compound-literal-1.c:8:15 punctuator )
compound-literal-1.c:8:17 punctuator {
compound-literal-1.c:8:19 constant 0
compound-literal-1.c:8:21 punctuator }
compound-literal-1.c:8:22 punctuator )
compound-literal-1.c:8:23 punctuator .
compound-literal-1.c:8:24 identifier i
compound-literal-1.c:8:26 punctuator +=
compound-literal-1.c:8:29 constant 1
compound-literal-1.c:8:30 punctuator ;
compound-literal-1.c:9:1 punctuator }
